ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/DefendantAssert.java

```diff
@@ -0,0 +1,221 @@
+package uk.gov.hmcts.cmc.ccd.assertion;
+
+import org.assertj.core.api.AbstractAssert;
+import uk.gov.hmcts.cmc.ccd.domain.defendant.CCDDefendant;
+import uk.gov.hmcts.cmc.domain.models.legalrep.Representative;
+import uk.gov.hmcts.cmc.domain.models.otherparty.CompanyDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.IndividualDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.SoleTraderDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPartyType.COMPANY;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPartyType.INDIVIDUAL;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPartyType.ORGANISATION;
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPartyType.SOLE_TRADER;
+
+public class DefendantAssert extends AbstractAssert<DefendantAssert, TheirDetails> {
```

Comment:
We have `uk.gov.hmcts.cmc.ccd.assertion.TheirDetailsAssert` in master for the same. Please use that.